app/javascript/components/server-components/Discover/index.tsx

```diff
@@ -148,9 +149,49 @@ const addInitialOffset = (params: SearchRequest) =>
     ? { ...params, from: recommendedProductsCount + 1 }
     : params;
 
+const BlackFridayButton = ({
```

Comment:
I had to make the CTA a React component as I need to dynamically update the CTA's URL everytime the taxonomy changes.
Might be worth extracting to a generic component later.